sources/packages/api/src-sql/EducationPrograms/Add-approval-columns.sql

```diff
@@ -0,0 +1,51 @@
+-- Add submitted_on to education_programs table.
+ALTER TABLE
+  sims.education_programs
+ADD
+  COLUMN IF NOT EXISTS submitted_on TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now();
+
+COMMENT ON COLUMN sims.education_programs.submitted_on IS 'Education program submitted date.';
+
+-- Add status_updated_on to education_programs table.
+ALTER TABLE
+  sims.education_programs
+ADD
+  COLUMN IF NOT EXISTS status_updated_on TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now();
+
+COMMENT ON COLUMN sims.education_programs.status_updated_on IS 'Education program status updated on';
+
+-- Add effective_end_date to education_programs table.
+ALTER TABLE
+  sims.education_programs
+ADD
+  COLUMN IF NOT EXISTS effective_end_date DATE;
```

Comment:
what is the purpose of effective_end_date? I saw this being entered in approve program modal.